README.rst

```diff
@@ -166,7 +167,23 @@ For each vocabulary that is created in the admin the following happens:
  - A persistent ``IVocabularyFactory`` is registered with the name
    ``collective.indexvocabularies.{index_name}``
  - A series of registry entries are created that register the widget as a
-   facet and filter for ``plone.app.querystring`
+   facet and filter for ``plone.app.querystring``
+
+
+The addon also subclasses the `default IAjaxSelect tag widget <https://github.com/plone/plone.app.z3cform/blob/master/plone/app/z3cform/interfaces.py#L82>`_
+- from `plone.app.z3cform <https://github.com/plone/plone.app.z3cform>`_ in
+order to provide supermodel import/export support. This could easily be added to
+``plone.app.z3cform`` which would remove the need for an additional widget.
+
+
+Alternatives
+------------
+
+Depending on your specific usecase you might be able to try the following:
+
+ - Create custom behaviors in your own addon
+ - Use the `plone.app.vocabularies.Catalog` vocabualary. See `this discussion <https://community.plone.org/t/widget-parameter-for-catalogsource-based-choicefield/18129/3>`_
```

Comment:
reStructuredText uses double backticks for inline literals, whereas Markdown uses single backticks.
```suggestion
 - Use the ``plone.app.vocabularies.Catalog`` vocabualary. See `this discussion <https://community.plone.org/t/widget-parameter-for-catalogsource-based-choicefield/18129/3>`_
```